src/CommentInfo/CommentInfo.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './CommentInfo.scss';
+
+type Props = {
+  comment: Comments,
```

Comment:
У тебя один коммент в пропсах, тип называется Comments - множественным числом
```suggestion
  comment: PostComment,
```